features/step_definitions/asset_host_steps.rb

```diff
@@ -1,25 +1,42 @@
 # frozen_string_literal: true
 
+Given /^the asset host is unset$/ do
+  Rails.configuration.three_scale.asset_host = nil
+end
+
 Given /^the asset host is set to "(.*)"$/ do |asset_host|
   cdn_url = "#{asset_host}:#{Capybara.current_session.server.port}"
-  Rails.configuration.asset_host = cdn_url
   Rails.configuration.three_scale.asset_host = cdn_url
 end
 
 Then /^((javascript|font)\s)?assets should be loaded from the asset host$/ do |asset_type|
-  cdn_url = Rails.configuration.asset_host
+  cdn_url = Rails.configuration.three_scale.asset_host
   js_regexp = Regexp.new("https?://#{cdn_url}/packs.*?\\.js")
   font_regexp = Regexp.new("https?://#{cdn_url}/packs.*?\\.eot")
 
   assert_not_nil Capybara.page.source.match js_regexp if ['javascript', nil].include? asset_type
   assert_not_nil Capybara.page.source.match font_regexp if ['font', nil].include? asset_type
 end
 
-Then /^assets shouldn't be loaded from the asset host$/ do
+Then /^((javascript|font)\s)?assets shouldn't be loaded from the asset host$/ do |asset_type|
   # When no CDN is set, we expect to find relative paths for fonts and js
   js_regexp = Regexp.new('src="/packs.*?\.js"')
   font_regexp = Regexp.new('url\(/packs.*?\.eot\)')
 
+  assert_not_nil Capybara.page.source.match js_regexp if ['javascript', nil].include? asset_type
+  assert_not_nil Capybara.page.source.match font_regexp if ['font', nil].include? asset_type
+end
+
+Then /^provided assets should be loaded from the asset host$/ do
+  cdn_url = Rails.configuration.three_scale.asset_host
+  js_regexp = Regexp.new("https?://#{cdn_url}/dev-portal-assets/.*?\\.js")
+
+  assert_not_nil Capybara.page.source.match js_regexp
+end
+
+Then /^provided assets shouldn't be loaded from the asset host$/ do
```

Comment:
`provided` assets is not very clear. Maybe a little bit better ame would be `portal` assets?

Reply:
I don't know how to call this. I mean specifically the assets that are provided by us, exclusively to be used by clients through the `cdn_asset` tag in their developer portals, but not being part of porta. I think `portal` assets is also confusing because it's too generic. Any other suggestion?